### DISCUSSION TITLE
VMX2 Template format error: YAML not well-formed. (line 28, column 66)

### DISCUSSION THREAD

**Discussion by SyndrixLane** [amazon-connect/amazon-connect-salesforce-scv#120] [Q&A] [answered]
Hello,

I am installing the voicemail express, I am at the process of deploying the cloud stack formation.
I downloaded the  vmx.yaml file 
https://github.com/amazon-connect/amazon-connect-salesforce-scv/blob/master/Solutions/VMX2-VoicemailExpress/CloudFormation/vmx.yaml
I then when to cloud formation and create stack, upload a template file, selected this file then clicked next.
I then get a red bar error at the top of the page that says
Template format error: YAML not well-formed. (line 28, column 66)

Can someone help me resolve?

**Comment by dougjaso** [comment, marked as answer]
Most likely, the file saved was not the actual YAML, but instead the HTML representation of it. end me an email at dougjaso@amazon.com and i can send it to you.

**Reply by SyndrixLane** [reply]
Thank you.

You are correct

The guide says 
Right-click/control-click to download the [Voicemail Express CloudFormation template](https://github.com/amazon-connect/amazon-connect-salesforce-scv/blob/master/Solutions/VMX2-VoicemailExpress/CloudFormation/vmx.yaml) and save it to your computer

Instead I followed the link and then on the "Raw" button I right clicked and save link as and this gave me the correct yaml file/formatting.

I am now able to proceed, thanks.
